Updated README and CONTRIBUTING to reflect sa-ibmi (#10)
IBM/sqlalchemy-ibmi: README.md

SQLAlchemy adapter for IBM i
=========

The IBM i SQLAlchemy adapter provides a [SQLAlchemy](https://www.sqlalchemy.org/) interface to Db2 for [IBM i](https://en.wikipedia.org/wiki/IBM_i).

**Please note that this project is still in active development and is not ready for use.** :rotating_light: 

```python
from sqlalchemy import create_engine, db

e = create_engine("db2+ibm_db://user:pass@host[:port]/database")
cnxn = e.connect()
metadata = db.Metadata()
table = db.Table('table_name', metadata, autoload=True, autoload_with=e)
query = db.select([table])
result = cnxn.execute(query)
result = result.fetchall()

print(result[0])

```

For more, check out the [samples](samples). (TODO)

Installation
-------------
TODO
 
Feature Support
----------------
TODO

Version
--------

TODO

Documentation
-------------

TODO

> Known Limitations in ibm_db_sa adapter for DB2 databases

Reviewer: ```suggestion
Known Limitations
```
Missed reference to ibm_db_sa

[![Build Status](https://travis-ci.com/IBM/sqlalchemy-ibmi.svg?branch=master)](https://travis-ci.com/IBM/sqlalchemy-ibmi)
[![Latest version released on PyPi](https://img.shields.io/pypi/v/sqlalchemy-ibmi.svg)]()
[![](https://img.shields.io/pypi/pyversions/sqlalchemy-ibmi.svg)]()

> [![Documentation Status](https://readthedocs.org/projects/sqlalchemy-ibmi/badge/?version=latest)]()

Reviewer: Since these don't exist, let's omit them. We can create issues to get them set up.

> SQLAlchemy-IBMi

Reviewer: ```suggestion
SQLAlchemy adapter for IBM i
```

=========

> The SQLAlchemy-IBMi adapter provides the Python/[SQLAlchemy](https://www.sqlalchemy.org/) interface to Db2 for [IBM i](https://en.wikipedia.org/wiki/IBM_i).

Reviewer: ```suggestion
The  IBM i SQLAlchemy adapter provides an [SQLAlchemy](https://www.sqlalchemy.org/) interface to Db2 for [IBM i](https://en.wikipedia.org/wiki/IBM_i).
```

Nice, I like the added links. I think we should refer to it as "the IBM i SQLAlchemy adapter" instead of camel-casing/titleizing the package name, though.

Reviewer: Can you also mention here that this project is experiencing a lot of changes/cleanups and is not ready for use. Using `:rotating_light:` might also be good.

4) UPDATE CASCADE for foreign keys not supported
5) DEFERRABLE INITIALLY deferred not supported
6) Subquery in ON clause of LEFT OUTER JOIN not supported

Credits
-------
- ibm_db_sa for SQLAlchemy was first produced by IBM Inc., targeting version 0.4.
- The library was ported for version 0.6 and 0.7 by Jaimy Azle.
- Port for version 0.8 and modernization of test suite by Mike Bayer.

> - Port for sqlalchemy-ibmi by Naveen Ram/Kevin Adler.

Reviewer: I think we should move this section to the end of the file.

> Contributing to sqlalchemy-ibmi python project

Reviewer: ```suggestion
Contributing to sqlalchemy-ibmi
```

----------------------------------------
Please read the [contribution guidelines](contributing/CONTRIBUTING.md).

```
The developer sign-off should include the reference to the DCO in remarks(example below):
DCO 1.1 Signed-off-by: Random J Developer <random@developer.org>
```

Releasing a New Version
-----------------------

Run the following commands

```bash
# checkout and pull the latest code from master
git checkout master
git pull

# bump to a release version (a tag and commit are made)
bumpversion release

# remove any old distributions
rm dist/*

# build the new distribution
python setup.py sdist

# bump to the new dev version (a commit is made)
bumpversion --no-tag patch

# push the new tag and commits
git push origin master --tags

# upload the distribution to PyPI
twine upload dist/*

> ```

Reviewer: We should probably omit this until we get #8 implemented.

License
-------

> TODO

Reviewer: ```suggestion
[Apache 2.0](LICENSE)
```
